### ISSUE TITLE
plotly-locale-nl-latest.js is not latest translations for `nl` locale

### ISSUE THREAD

**Issue by eirvandelden** [plotly/plotly.js#6399]
What I did:
- include `<script src="https://cdn.plot.ly/plotly-locale-nl-latest.js"></script>` to get latest nl locale

What I expected:
- The translations like in `https://github.com/plotly/plotly.js/blob/master/lib/locales/nl.js` with translations in the dictionary part and updates to dayMonth

What I got:
- Only the days and months are translated. Actual content hosted now:
```js
var locale={moduleType:"locale",name:"nl",dictionary:{},format:{days:["zondag","maandag","dinsdag","woensdag","donderdag","vrijdag","zaterdag"],shortDays:["zon","maa","din","woe","don","vri","zat"],months:["januari","februari","maart","april","mei","juni","juli","augustus","september","oktober","november","december"],shortMonths:["jan","feb","maa","apr","mei","jun","jul","aug","sep","okt","nov","dec"],date:"%d-%m-%Y"}};"undefined"==typeof Plotly?(window.PlotlyLocales=window.PlotlyLocales||[],window.PlotlyLocales.push(locale)):Plotly.register(locale);
```

I think what is hosted on https://cdn.plot.ly/plotly-locale-nl-latest.js is not the latest, but a version before my changes were merged and released in 2.14.0: #6261

**Comment by alexcjohnson** [comment]
We stopped updating `latest` when we released v2.0, to avoid accidental breaking changes

**Comment by eirvandelden** [comment]
Ooooh, I hadn't realized that was for both the library and translations 🙈 

Thanks for commenting!
